File: MeetupBackend/Models/Event.cs
namespace MeetupBackend.Models
{
    public class Event
    {
        public string? Id { get; set; }
        public string Title { get; set; } = string.Empty;
        public string Description { get; set; } = string.Empty;
        public DateTime Date { get; set; }
        public string Category { get; set; } = string.Empty;
        
        public double Latitude { get; set; }
        public double Longitude { get; set; }
    }
}

File: meetupfrontend/src/models/Event.ts
export interface MeetupEvent {
  id: string;
  title: string;
  description: string;
  latitude: number;
  longitude: number;
  date: string;
  category: string; 
  attendees: string[]; 
  creatorId: string;   
  
  // Optional fields
  distanceKm?: number;
  friendsGoing?: number;
}

export interface CreateEventDto {
  title: string;
  description: string;
  date: string;     
  category: string;
  latitude: number;
  longitude: number;
}

export interface UpdateEventDto {
  title?: string;
  description?: string;
  date?: string;
  category?: string;
  latitude?: number;
  longitude?: number;
}